x-pack/plugins/apm/server/routes/settings/agent_configuration/get_config_applied_to_agent_through_fleet.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { termQuery, rangeQuery } from '@kbn/observability-plugin/server';
+import datemath from '@kbn/datemath';
+import { METRICSET_NAME } from '../../../../common/elasticsearch_fieldnames';
+import { Setup } from '../../../lib/helpers/setup_request';
+
+export async function getConfigsAppliedToAgentsThroughFleet({
+  setup,
+}: {
+  setup: Setup;
+}) {
+  const { internalClient, indices } = setup;
+
+  const params = {
+    index: indices.metric,
+    size: 0,
+    body: {
+      query: {
+        bool: {
+          filter: [
+            ...termQuery(METRICSET_NAME, 'agent_config'),
+            ...rangeQuery(
+              datemath.parse('now-1m')!.valueOf(),
```

Comment:
I think 15m was fine honestly. 1m is a little risky when clock drift, ingestion delay and the interval at which APM Server sends these metricsets come into play.

Reply:
Changed it back to 15m